canonical-web-and-design/ubuntu.com
# Fix Issue #9306 - UC20 only work with UEFI.

The required UEFI features is provided by OVMF.

## Done

- Updated the instructions to run UC20 with KVM. 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See the result in http://0.0.0.0:8001/download/kvm

## Issue / Card

Fixes #9306
